### PR TITLE
Final database import

### LINKED ISSUE #45: Figure out how to import MongoDB data into PostgreSQL
Using the JSON files provided by Jeo

### PR DESCRIPTION
**Pull request by rer7891**
### Sidebar Checklist

- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project

### Issues Resolved
Resolves #45 

### Problem Addressed
Need to import old database information to local and then push to heroku
### Solution Implemented
Imports old database information to local and then pushed to Heroku database

### CODE REVIEW

File: bin/rake [hunk 1]
```diff
@@ -1,4 +1,8 @@
-#!/usr/bin/env ruby
+begin
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError => e
+  raise unless e.message.include?('spring')
+end
```
Comment (DanielEFrampton) [inline bin/rake]: Does it work without this?

File: db/migrate/20200226175936_remove_oid_from_users.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveOidFromUsers < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :users, :oid, :string
+  end
+end
```
Comment (DanielEFrampton) [inline db/migrate/20200226175936_remove_oid_from_users.rb]: I'm still confused about why we need to do this but let's roll with it 😆

File: lib/tasks/import.rake [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        pairee_id = User.where(oid: pairing_data["pairee"]["$oid"]).first.id
+      end
+    Pairing.create( pairer_id: pairer_id,
+                    pairee_id: pairee_id,
+                    date: pairing_data["date"],
+                    time: pairing_data["time"],
+                    notes: pairing_data["notes"]
+                   )
+  end
+  ActiveRecord::Migration.remove_column :users, :oid, :string
```
Comment (DanielEFrampton) [inline lib/tasks/import.rake]: Let's comment this out as well.

File: lib/tasks/import.rake [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  File.open(ARGV[2]).each do |line|
+    pairing_data = JSON.parse(line)
+      if User.where(oid: pairing_data["pairer"]["$oid"]).empty?
+        pairer_id = nil
+      else
+        pairer_id = User.where(oid: pairing_data["pairer"]["$oid"]).first.id
+      end
+
+      if pairing_data["pairee"].nil?
```
Comment (Capleugh) [inline lib/tasks/import.rake]: Really creative solution! Love it.

File: bin/rake [hunk 5]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env ruby
```
Comment (DanielEFrampton) [inline bin/rake]: I think we might need this after all; looked it up and the `#!` thing is a "shebang" and actually executes stuff. https://en.wikipedia.org/wiki/Shebang_(Unix)

**Pull request merged**